__tests__/force/Gravity_test.re

```diff
@@ -0,0 +1,53 @@
+open Jest;
+
+let it = test;
+
+describe("Gravity", () => {
+  describe("force", () => {
+    it("should derive the force of gravity between two objects", () =>
+      Expect.(
+        expect(Gravity.force(~m1=1., ~m2=1., ~r=1.))
+        |> toBeCloseTo(Gravity.g)
+      )
+    );
+
+    it(
+      "should correctly derive the force of gravity at the Earth's surface", () => {
+      let massEarth = 5.97 *. Math.pow(~base=10, ~exp=24);
+      let massObject = 25.;
+      let rEarth = 6.371 *. Math.pow(~base=10, ~exp=6);
+      Expect.(
+        expect(
+          Gravity.force(~m1=massEarth, ~m2=massObject, ~r=rEarth)
+          |> Printf.printf("%0.*f\n", 2),
```

Comment:
A little trick from OCaml – this more or less mimics the behavior of JS `Number.toFixed()`, where you can round a floating point number to a specific precision. When originally calculating this out, I was getting a number like 2.45283472e24. TIL that `.toFixed()` won't actually truncate numbers w/ greater than 21 places of precision 😂 :mild-panic: